packages/augur-tools/src/templates-source.ts

```diff
@@ -1392,6 +1426,9 @@ export const TEMPLATES = {
                     {
                       text: `If the game is NOT played or is not deemed an official game, meaning, less than 90% of the scheduled match had been completed, the market should resolve as "Unofficial game/Cancelled".`,
                     },
+                    {
+                      text: 'This market is intended to be about a Single Group, if this is not the case, this market should settle as invalid.'
```

Comment:
Single group doesnt makes sense here. Its a sports market team makes more sense.

Reply:
ok updated to teams